waspc/src/Wasp/Generator/FileDraft.hs

```diff
@@ -82,3 +86,7 @@ createCopyDirFileDraft dstPath srcPath =
 createTextFileDraft :: Path' (Rel ProjectRootDir) File' -> Text -> FileDraft
 createTextFileDraft dstPath content =
   FileDraftTextFd $ TextFD.TextFileDraft {TextFD._dstPath = dstPath, TextFD._content = content}
+
+createJsonFileDraft :: Path' (Rel ProjectRootDir) File' -> Aeson.Value -> FileDraft
```

Comment:
Should we really call it this way, if there is no JsonFileDraft? Maybe we could instead call it `createBytesFileDraftFromJson`?